client/src/browser-table.html

```diff
@@ -263,7 +263,15 @@ <h1 label>Browser support</h1>
         </a>
       </div>
       <div class="row">
-        <div label="Imports"><svg viewBox="0 0 26 22"><use xlink:href="/sprite.octicons.svg#import"></use></svg></div>
+        <div label='<script type="module">'><svg viewBox="0 0 26 22"><use xlink:href="/sprite.octicons.svg#import"></use></svg></div>
```

Comment:
Why not `ES modules` o something similar?
There are not `<template>` or `<link rel="import">` in the others.

Reply:
Because there's multiple parts that are separately tracked. eg. [dynamic import](https://www.chromestatus.com/feature/5684934484164608) is listed separately. We'll probably include that later (its hard to find info on it and its not very interesting to show in the table yet).

Eventually we'll replace this with one row of "ES Modules".

Reply:
Does that make sense? @abdonrd

Reply:
Hmm wonder if the row of ES Modules with the row containing support for the parts? Modules, nomodule, and dynamic imports?

Reply:
Hmm, if we did that, it would have to be its own table with 3 rows.

Reply:
There's also worker modules. I created https://aremodulesready.com for this purpose :)

Reply:
@samuelli got it! Makes sense.

@stramel @matthewp thanks for the feedback!